.gitmodules

```diff
@@ -1,5 +1,4 @@
-
 [submodule "owncloud-android-library"]
 	path = owncloud-android-library
-	url = git://github.com/owncloud/android-library.git
-	branch = develop
+	url = https://github.com/owncloudChalmers/android-library.git
```

Comment:
Wrong URL here; I guess it cannot be fixed until the PR in the library is merged...